config/locales/document_sets/document_sets-es.yml

```diff
@@ -2,52 +2,84 @@
+      document_set_image_description: Una imagen que se utilizará para ilustrar la descripción del conjunto de documentos.
+      here: aquí
+      no_image: Sin imagen
+      save_changes: Guardar cambios
       title: Título
+      upload_image: Subir imagen
       url: URL
+      url_description: Utilice letras minúsculas y guiones entre las palabras.
+    edit_privacy:
+      document_set_collaborators: Colaboradores del conjunto de documentos
+      document_set_privacy: 'Conjunto de documentos de privacidad: <i>%{privacy}</i>'
+      document_set_restricted_description: El conjunto de documentos solo puede ser visto por los colaboradores que se enumeran a continuación. Puede permitir que la colección sea de lectura pública.
+      document_set_unrestricted_description: Cualquier persona en Internet puede ver el conjunto de documentos. Cualquier obra incluida en un conjunto de documentos público será legible, incluso si la colección principal es privada. Puede hacer que el conjunto de documentos sea privado para restringir su visibilidad a los colaboradores.
+      edit_collaborators: Editar colaboradores
+      make_document_set_private: Hacer que el conjunto de documentos sea privado
+      make_document_set_public: Hacer público el conjunto de documentos
+      private: Privado
+      public: Público
     edit_set_collaborators:
       remove_collaborator: Quitar colaborador
       set_collaborators: Colaboradores del conjunto de documentos
       set_collaborators_description: Los colaboradores pueden transcribir y editar conjuntos de documentos privados.
+    edit_tabs:
+      general: General
+      manage_works: Gestionar obras
+      privacy_access: Privacidad y acceso
+    edit_works:
+      add: Agregar
+      add_work_title_to_document_set: Agregar %{work_title} al conjunto de documentos
+      document_is_marked_public: Si el conjunto de documentos está marcado como público, las obras incluidas en él serán legibles incluso si la colección está marcada como privada.
+      n_pages:
+        one: '1 página:'
+        other: "%{count} páginas: pl"
+      save: Ahorrar
+      search: Buscar
+      search_for_works: Buscar obras...
+      status: Estado
+      work: Obra
     index:
-      actions: Acciones
-      aggregating_works_thematic_exhibits: Agregando obras a exposiciones temáticas
-      assign_works_to_document_sets: Asignar Obras a Conjuntos de Documentos
-      create_a_document_set: Crear un Conjunto de Documentos
-      delete: Eliminar
-      delete_document_set_confirm_message: "¿De verdad deseas eliminar este conjunto de documentos? Esto eliminará tu conjunto de documentos, pero no borrará ninguna de las obras en el conjunto de documentos."
-      document_sets_are_sub_sets: 'Los conjuntos de documentos son subconjuntos de los documentos dentro de una colección. Tienen varios usos, incluyendo:'
-      document_sets_for: Conjuntos de Documentos para %{title}
+      actions: Comportamiento
```

Comment:
Similarly, `Acciones` is turned to `Comportamiento`, indicating a retranslation.